app/Models/Competition.php

```diff
@@ -31,16 +31,19 @@ public function users()
     /**
      * Gets a CSV export of all the users in this competition.
      *
+     * @param \Gladiator\Repositories\UserRepositoryContract $repository
      * @return \League\Csv\ $csv
      */
-    public function getCSVExport()
+    public function getCSVExport($repository)
```

Comment:
well then i might as well move the entire thing into the controller, but i thought we didn't want to clutter the controllers up? i dunno what our best practices are for this stuff